packages/devtools/lib/src/memory/memory.dart

```diff
@@ -58,16 +58,10 @@ class MemoryScreen extends Screen with SetStateMixin {
 
   void updateResumeButton({@required bool disabled}) {
     resumeButton.disabled = disabled;
-    resumeButton.changeIcon(disabled
```

Comment:
our CSS is applying an opacity of 0.5 for disabled so this wasn't needed, right?

Reply:
Correct!